dataland-frontend/src/components/forms/CreateSFDRDataset.vue

```diff
@@ -0,0 +1,330 @@
+<template>
+  <Card class="col-12 page-wrapper-card p-3">
```

Comment:
This component has really a lot of duplicate code from "CreateLksgDataset".

I suggest that we introduce a "CreateDatasetBase" component which covers the basic stuff and has a slot to integrate custom stuff.

Then "CreateLksg" and "CreateSfdr" can just use "CreateDatasetBase" and add their specific stuff.

What do you think?

Reply:
Sounds good to me. I would propose to do this once @MarcinStanczyk7 is finished working on this to avoid having severe merge conflicts.

Reply:
Yes agree

Reply:
in the middle of progress